packages/suite/src/components/guide/GuideHint.tsx

```diff
@@ -29,25 +30,41 @@ const REGEX = new RegExp(`^(${BULB_EMOJI}|${WARNING_EMOJI})\\s*`);
 // The displayed variant is determined by an emoji at the start of the markdown quote.
 export const GuideHint = ({ children }: React.BlockquoteHTMLAttributes<HTMLQuoteElement>) => {
     // We dig the message from children to check for an emoji indicating the component variant.
-    // There should be three children, with the first and the last being just newline characters.
+    // There should be three children, with the first and the last being just newline characters - we filter them out as invalid elements.
     // The middle one is a ReactElement whose children are the content of the message (array of strings and ReactElements in case of links).
```

Comment:
```suggestion
    // The middle one is a ReactElement whose children are the content of the message (array of strings and ReactElements in case XXX).
```
XXX - not only links, but also `<strong>`, and probably others

Reply:
39d96773a0cc3c2097c6ee82a67ca342fc29d5da